package.json

```diff
@@ -207,7 +207,7 @@
     "test": "standard && npm run test-tap",
     "test-coverage": "npm run tap-cover -- \"test/tap/*.js\" \"test/network/*.js\"",
     "test-tap": "npm run tap -- \"test/tap/*.js\"",
-    "test-node": "\"$NODE\" \"node_modules/.bin/tap\" --timeout 240 \"test/tap/*.js\" \"test/network/*.js\""
+    "test-node": "tap --timeout 240 \"test/tap/*.js\" \"test/network/*.js\""
```

Comment:
I wonder if we could do: 
```
"%NODE%" "node_modules/tap/bin/run.js"
```
on Windows? Will `cmd` do env var substitution for the command part?

If that will work I'd be inclined to have a `test-node-win` target with the above.